### DISCUSSION TITLE
How to avoid Minio POD drift in K8s

### DISCUSSION THREAD

**Discussion by llchry** [minio/minio#19807] [Q&A] [answered]
Assume that the Minio PODs are deployed as follows:
minio-0 on node1,
minio-1 on node2.
minio-2 on node3.

When the HELM changes and the POD is restarted at the same time, the distribution after the HELM upgrade may be as follows:
minio-0 on node2
minio-1 on node3
minio-2 on node1

At such times, Minio becomes unavailable. How can I avoid this?

**Comment by harshavardhana** [comment, marked as answer]
provide a local pv for correct affinity.
